### PR TITLE
Transition to new Upload instance

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

File: dns/prx.org-hosted_zone.yml [hunk 1]
```diff
@@ -378,11 +378,6 @@ Resources:
           TTL: "300"
           Type: CNAME
           Name: !Sub metrics.${Domain}
-        - ResourceRecords:
-            - upload.prx.tech.
-          TTL: "300"
-          Type: CNAME
-          Name: !Sub upload.${Domain}
```
Comment (farski) [inline dns/prx.org-hosted_zone.yml]: This has already been deployed

File: stacks/serverless/upload.prx.org.yml [hunk 2]
```diff
@@ -99,8 +101,8 @@ Resources:
   UploadRestApiStage:
     Type: "AWS::ApiGateway::Stage"
     Properties:
-      StageName: !If [CreateProductionResources, prod, stag] # TODO these shouldn't be the same
-      Description: !Sub "{EnvironmentType} stage for upload lambda gateway integration"
+      StageName: !If [CreateProductionResources, prod, stag]
+      Description: !Sub "${EnvironmentType} stage for upload lambda gateway integration"
```
Comment (kookster) [inline stacks/serverless/upload.prx.org.yml]: good catch $

File: stacks/serverless/upload.prx.org.yml [hunk 3]
```diff
@@ -110,34 +112,34 @@ Resources:
...
-  #           HostedZoneId: Z2FDTNDATAQYW2
+  UploadRestApiDomainName:
+    Type: "AWS::ApiGateway::DomainName"
+    Properties:
+      CertificateArn: !Ref WildcardCertificateArn
+      DomainName: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
+  UploadRestApiBasePathMapping:
+    Type: "AWS::ApiGateway::BasePathMapping"
+    Properties:
+      # BasePath: 'prod'
```
Comment (kookster) [inline stacks/serverless/upload.prx.org.yml]: Do we need this comment in there?

File: stacks/serverless/upload.prx.org.yml [hunk 4]
```diff
@@ -110,34 +112,34 @@ Resources:
...
+    Type: "AWS::Route53::RecordSetGroup"
+    Properties:
+      Comment: Record sets for upload signing API
+      HostedZoneName: prx.tech.
+      RecordSets:
+        - Type: AAAA
+          Name: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
+          AliasTarget:
+            DNSName: !GetAtt UploadRestApiDomainName.DistributionDomainName
+            HostedZoneId: Z2FDTNDATAQYW2
```
Comment (kookster) [inline stacks/serverless/upload.prx.org.yml]: We use `Z2FDTNDATAQYW2` in about 9 places in the templates, I wonder if it should be factored into a var of some kind?

**Pull request merged**